girderformindlogger/api/v1/invitation.py

```diff
@@ -196,8 +195,5 @@ def declineInvitation(self, invitation):
             raise AccessException(
                 "You must be logged in to accept an invitation."
```

Comment:
Small typo here, should say decline or remove instead of accept

Reply:
@henryrossiter good catch! https://github.com/ChildMindInstitute/mindlogger-app-backend/pull/227/commits/b18885b864140aea4a2abc70b2fcbe86b9a06cd2 gets rid of the log in requirement for declining. I think we don't want to force people to create an account just to decline, right?

Reply:
That works as well. 

I'm not sure I understand what the delete call does in the first place though. Even after a user declines an invitation, he/she could still view the invitation by clicking the same link, correct? Then he/she could choose whether to accept or decline again?

Reply:
Nope. The delete call deletes the invitation.